clvm/types.py

```diff
@@ -0,0 +1,15 @@
+from typing import Any, Callable, Dict, Tuple, Union
+
+
+CLVMAtom = Any
+CLVMPair = Any
+
+CLVMObjectType = Union["CLVMAtom", "CLVMPair"]
```

Comment:
I know I originally wrote this file, but I'm wondering if maybe we should remove `CLVMObjectType` since it's not actually used anywhere (and maybe other unused things in this file). It doesn't seem that mypy's typing system is advanced enough to do what we need to do.